.github/workflows/ci.yaml

```diff
@@ -0,0 +1,62 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - release
+  pull_request:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7', '3.8']
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
```

Comment:
I was initially surprised that this step takes ~1 minute with GitHub Actions compared to ~15-30 seconds with Travis CI. I see now that this isn't a fair comparison because we use conda-forge and install mamba here. It looks like we could consider using [package caching](https://github.com/marketplace/actions/setup-miniconda#caching-packages) in the future, if we wanted to speed this up a bit.

Reply:
Good idea, we can try that in the future to see how much it helps.